index.html

```diff
@@ -13,7 +13,21 @@
 </head>
 
 <body>
+  <footer>
+    <div class="botoes">
 
+    </div>
+
+    <div class="componenteTempo">
+      <p class="musicaFooter">Rihanna - Pon de Replay</p>
+      <div class="barras">
+        <div class="barraExterna">
+          <div class="barraInterna"></div>
```

Comment:
Vamos patronizar nomes de classes separados por traços, no css
```suggestion
          <div class="barra-interna"></div>
```